src/backend/joanie/tests/core/api/order/test_submit_installment_payment.py

```diff
@@ -486,7 +486,7 @@ def test_api_order_submit_installment_payment_with_credit_card_id_payload(
             },
         )
 
-        self.assertTrue(_mock_create_one_click_payment.called)
+        self.assertTrue(mock_create_one_click_payment.called)
```

Comment:
This is redundant, you are already testing that is method has been called with `assert_called_once_with`

```suggestion
```